src/metamodels/cgpm_metamodel.py

```diff
@@ -65,6 +65,8 @@
 );
 '''
 
+QUOTES = ['', '\'\'', '""', '\'"', '"\'']
```

Comment:
@curlette The `QUOTES` list should be identical to the one from the original review message:
https://github.com/probcomp/bayeslite/pull/523#discussion_r98938768

The ones above will not produce the desired behavior, consider the following test case
```python
+def test_empty_strings():
+    QUOTES = ['\'\'', '"\'\'"', '\'""\'', '\'"', '"\'']
+    bdb = bayeslite.bayesdb_open(builtin_metamodels=False)
+    bdb.sql_execute('CREATE TABLE t(x NUMERIC, y TEXT)')
+    for i, q in enumerate(QUOTES):
+        bdb.sql_execute('INSERT INTO t VALUES (?, ?)', (i, q,))
+    for i, q in enumerate(QUOTES):
+        cursor = bdb.sql_execute('''
+            SELECT x FROM t WHERE y = ?
+        ''', (q,))
+        assert next(cursor)[0] == i
+    cursor = bdb.sql_execute('SELECT * FROM t WHERE y IN (?,?,?,?,?)', QUOTES)
+    assert len(cursor.fetchall()) == len(QUOTES)
```